test/cpp/util/test_credentials_provider.h

```diff
@@ -29,6 +29,9 @@ namespace grpc {
 namespace testing {
 
 const char kInsecureCredentialsType[] = "INSECURE_CREDENTIALS";
+// Using kTls12CredentialsType and kTLS13CredentialsType forces the TLS version
```

Comment:
Suggested edit:

`The kTls12CredentialsType (resp. kTls13CredentialsType) credential type yields SSL credentials that only use TLS 1.2 (resp. 1.3).`

For the comment on `kTlsCredentialsType`, please move it so that it is directly above the declaration of the `kTlsCredentialsType` variable.